packages/oidc-auth/package.json

```diff
@@ -9,10 +9,10 @@
     "dist"
   ],
   "scripts": {
-    "test": "vitest",
-    "build": "tsup ./src/index.ts --format esm --dts",
-    "prerelease": "yarn build && yarn test",
-    "release": "yarn prerelease && yarn npm publish"
+    "build": "tsup ./src/index.ts --format esm",
+    "prepack": "yarn build",
+    "publint": "attw --pack --profile esm-only && publint",
+    "test": "vitest"
```

Comment:
This is the first package that I've noticed is esm only 👀

Reply:
I think exporting only esm is not intended. So can you make it export cjs as well?

Reply:
Sure thing 👍🏻